From b9ef7811512cd530a208a39d21d37870dabad511 Mon Sep 17 00:00:00 2001
From: Bogdan Hamciuc <bogdan.hamciuc@nxp.com>
Date: Tue, 2 Jun 2020 15:16:30 +0300
Subject: [PATCH 129/269] s32g: Add fiptool dependency on bl2

In some build configurations, bl2.bin may not be available when fiptool
is invoked. Add explicit dependency on bl2 to avoid such problems, most
notable in Yocto builds.

Upstream-Status: Pending 

Signed-off-by: Bogdan Hamciuc <bogdan.hamciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
Issue: ALB-5027
---
 plat/s32g/platform.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/s32g/platform.mk b/plat/s32g/platform.mk
index b9c69087c..3071b13fe 100644
--- a/plat/s32g/platform.mk
+++ b/plat/s32g/platform.mk
@@ -91,7 +91,7 @@ check_dtc_version:
 
 FIP_ALIGN := 512
 all: add_bl2_to_fip
-add_bl2_to_fip: fip
+add_bl2_to_fip: bl2 fip
 	${Q}${FIPTOOL} update ${FIP_ARGS} \
 		--tb-fw ${BUILD_PLAT}/bl2.bin \
 		${BUILD_PLAT}/${FIP_NAME}
-- 
2.17.1

